src/XRP/xrp-client-interface.ts

```diff
@@ -1,11 +1,15 @@
 import { Wallet } from 'xpring-common-js'
 import { BigInteger } from 'big-integer'
 import TransactionStatus from './transaction-status'
+import XRPLNetwork from '../Common/xrpl-network'
 
 /**
  * An interface describing XRPClient.
  */
 export default interface XRPClientInterface {
+  /** The XRPL Network of the node that this client is communicating with. */
```

Comment:
should this be `@param` like it is in `PayIDClient`? (or however they should be consistent)

Reply:
In payID client it is an actual parameter to the constructor. We use some syntactic sugar to synthesize a property. 

So when I write:
```
constructor(public readonly network: XRPLNetwork) {}
```

This expands to:
```
public readonly network: XRPLNetwork

constructor(network: XPRLNetwork) {
  this.network = network
}
```

The iVar declaration is what this interface is covering. So no need to @param this since it is just a variable that needs to exist.